demosplan/DemosPlanCoreBundle/Logic/Import/Statement/StatementFromRowBuilderWithZipSupport.php

```diff
@@ -155,6 +155,9 @@ protected function handleFileReferences(): ?ConstraintViolationListInterface
                     $fileContainer,
                     [new Type(FileContainer::class), new NotNull()]
                 );
+
+                $fileString = $fileContainer?->getFileString();
```

Comment:
may be we need to right a little comment here to explain why it is necessary to set the files at this point, something like 'the files have to be copied later from the generated original Statement object that's why they have to be set at this point otherwise they will be missing in new generated statement'